bazel/google_cloud_cpp_deps.bzl

```diff
@@ -130,24 +130,24 @@ def google_cloud_cpp_deps(name = None):
         name = "boringssl",
         # Use github mirror instead of https://boringssl.googlesource.com/boringssl
```

Comment:
maybe s/github/a/ ?

Reply:
I don't know ... "github mirror" seems strictly more informative than "a mirror" when referring to https://github.com/google/boringssl.

Reply:
Move comment location to make it clearer.  Thanks.